File: entity-recognition/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingEntityRecognition.java
NaturalLanguageUnderstanding service = new NaturalLanguageUnderstanding("2018-03-16", username, password);

EntitiesOptions options = new EntitiesOptions.Builder().build();

Features features = new Features.Builder()
    .entities(options)
    .build();

AnalyzeOptions parameters = new AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build();

AnalysisResults response = service
    .analyze(parameters)
    .execute();

List<EntitiesResult> entities = response.getEntities();

File: entity-recognition/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingEntityRecognition.kt
// "text" is a string containg the input text.

val service = NaturalLanguageUnderstanding("2018-03-16", username, password)

val options = EntitiesOptions.Builder().build()

val features = Features.Builder()
    .entities(options)
    .build()

val parameters = AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build()

val response = service
    .analyze(parameters)
    .execute()

val entities = response.entities
